docs/rackhd/npm_based_installation.rst

```diff
@@ -0,0 +1,242 @@
+
+NPM Based Installation
+---------------------------------
+
+
+Ubuntu
+~~~~
+Prerequisites
+^^^^
+
+**NICs**
+
+1. **UBUNTU 14.04**
+
+* Start with an Ubuntu trusty(14.04) instance with 2 nics:
+
+   * eth0 for the 'public' network - providing access to RackHD APIs, and providing routed (layer3) access to out of band network for machines under management
+
+   * eth1 for dhcp/pxe to boot/configure the machines
+
+* Edit the network:
+
+   * eth0 - assign IP address as appropriate for the environment, or you can use DHCP
+
+   * eth1 static ( 172.31.128.0/22 )
+
+     this is the 'default'. it can be changed, but more than one file needs to be changed.)
+
+
+####
+
+2. **UBUNTU 16.04**
+
+* Start with an Ubuntu xenial(16.04) instance with 2 nics:
+
+   * ens160 for the 'public' network - providing access to RackHD APIs, and providing routed (layer3) access to out of band network for machines under management
+
+   * ens192 for dhcp/pxe to boot/configure the machines
+
+* Edit the network:
+
+   * ens160 - assign IP address as appropriate for the environment, or you can use DHCP
+
+   * ens192 static ( 172.31.128.0/22 )
+
+     this is the 'default'. it can be changed, but more than one file needs to be changed.)
+
+**Packages**
+
+1. **NodeJS 4.x**
+
+   **If Node.js 4.x is not installed**
+
+* **Remove Node.js (< 4.0)**
+
+   *If Node.js is installed via apt, but is older than version 4.x, do this first* (apt-get installs v0.10 by default)
+   .. code::
```

Comment:
Need an empty line before `.. code::`, otherwise you will see the render content format is not as expected.